distrobox-create

```diff
@@ -379,8 +379,6 @@ generate_command() {
 	# Mount also the distrobox-init utility as the container entrypoint.
 	result_command="${result_command}
 		--env \"SHELL=${SHELL:-"/bin/bash"}\"
-		--env \"HOME=${container_user_home}\"
-		--volume \"${container_user_home}\":\"${container_user_home}\":rslave
```

Comment:
Both of these can be done later, since there is a new conditional following.  HOME will not be set and then set again (overridden) later.